package/gluon-core/files/lib/gluon/upgrade/001-reset-uci

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+NETWORK_CFG='/etc/config/network'
+NETWORK_SAVED="${NETWORK_CFG}_gluon-old"
+
+SYSTEM_CFG='/etc/config/system'
+SYSTEM_SAVED="${SYSTEM_CFG}_gluon-old"
+
+# Save old configs (unless there is already a saved config,
+# which means that the previous upgrade was interrupted)
+if [ -s "$NETWORK_CFG" ] && ! [ -s "$NETWORK_SAVED" ]; then
+	mv -f "$NETWORK_CFG" "$NETWORK_SAVED"
+fi
+if [ -s "$SYSTEM_CFG" ] && ! [ -s "$SYSTEM_SAVED" ]; then
+	mv -f "$SYSTEM_CFG" "$SYSTEM_SAVED"
+fi
+
+# Generate a new network config
+rm -f /etc/board.json "$NETWORK_CFG" "$SYSTEM_CFG"
+config_generate
```

Comment:
This command doesn't seem to be silent... 

```
root@wgtest-1043-lemoer:~# gluon-reconfigure 
Configuring: 001-reset-uci
cfg030f15
cfg073777
cfg081ec7
cfg091ec7
cfg01e48a
Configuring: 002-migrate-system
Configuring: 005-set-domain
Configuring: 005-wireless-migration
...
```

Can we silence it?

Reply:
We can redirect the output to `/dev/null`. I'm not sure if the script ever prints something interesting that might help with debugging though.